### ISSUE TITLE
sorting option missing

### ISSUE THREAD

**Issue by lollita** [aVadim483/fast-excel-templator#7]
I’m experiencing a strange issue in an Excel file create with fast-excel-templator.
When I open the file, the sorting option is missing.
However, if I switch to another sheet and then return to the original one, the sorting option appears and works fine.
Do it is related to fast-excel-templator?

**Comment by aVadim483** [comment]
I need an example to test

**Comment by lollita** [comment]
It is related.
[Here](https://interno.altervista.org/temp/example.zip) is a example.
I not know because also it put today date in A1.

**Comment by lollita** [comment]
Have you been able to reproduce the problem?

**Comment by aVadim483** [comment]
I looked at your example but I don't understand what the problem is, I don't see sorting options on the sheets you write about even if I switch many times.

By the way, your code in version 2 will look like this:
```php
$rowTemplate = $sheet->getRowTemplate(1);
foreach ($rows as $v){
    var_dump($v);
	$sheet->insertRow($rowTemplate, $v);
}
```

**Comment by lollita** [comment]
See result.xlsx.
I maked a [video](https://interno.altervista.org/temp/RESULT_NoSortXlsx.mp4).

**Comment by lollita** [comment]
Have you been able to reproduce the problem?

**Comment by aVadim483** [comment]
Yes, but I don't know the reason yet

**Comment by aVadim483** [comment]
fixed in v2.1.1
